examples/cluster/plot_coin_ward_segmentation.py

```diff
@@ -36,7 +36,7 @@
 # Applying a Gaussian filter for smoothing prior to down-scaling
 # reduces aliasing artifacts.
 smoothened_coins = gaussian_filter(orig_coins, sigma=2)
-rescaled_coins = rescale(smoothened_coins, 0.2, mode="reflect")
+rescaled_coins = rescale(smoothened_coins, 0.2, mode="reflect",multichannel=False,anti_aliasing=False)
```

Comment:
Add whitespace after each comment. Then split the line so it doesn't exceed 80 char